clcache.py

```diff
@@ -73,6 +73,13 @@ def filesBeneath(path):
             yield os.path.join(path, filename)
 
 
+def subdirectories(path, absolute=True):
```

Comment:
From the name it is not clear to me, if this is recursive or not. What about `directChildDirectories`?